imageset.py

```diff
@@ -33,6 +38,20 @@
 )
 
 
+def _expand_template(template, indices):
+    # type: (str, Iterable[int]) -> List[str]
```

Comment:
```suggestion
def _expand_template(template: str, indices: Iterable[int]) -> List[str]:
```
typing syntax is probably more helpful (untested, not sure I'm doing this right)